inugami_webapp/src/main/webapp/js/app/components/providers/gitlab_merge_requests/gitlab.merge.requests.component.ts

```diff
@@ -0,0 +1,40 @@
+// ANGULAR CORE MODULES --------------------------------------------------------
+import {Component, Input, AfterContentInit}     from '@angular/core';
+
+@Component({
+    selector: 'gitlab-merge-requests',
+    templateUrl: "js/app/components/providers/gitlab_merge_requests/gitlab.merge.requests.component.html"
+})
+export class GitlabMergeRequestsComponent implements AfterContentInit {
+    /**************************************************************************
+    * ATTRIBUTES
+    **************************************************************************/
+    @Input() styleClass                     : string;
+    @Input() event                          : string;
+    @Input() pluginName                     : string;
+
+    private data                            : any[];

```

Comment:
Comme une note. 

Cela peut être problématique en cas d'AOP
https://stackoverflow.com/questions/34574167/angular2-should-private-variables-be-accessible-in-the-template

Mais pour le moment cela marche bien.

Reply:
Yes, it come from T plugin. Currently data also doesn't define any type. We can change visibility for the 1.0.1 realse ans create US for cleaning more this component